packages/mdc-chips/_variables.scss

```diff
@@ -14,16 +14,18 @@
 // limitations under the License.
 //
 
+@import "@material/theme/variables";
+
 $mdc-chip-border-radius-default: 16px;
-$mdc-chip-fill-color-default: rgba(black, .08);
-$mdc-chip-ink-color-default: rgba(black, .6);
+$mdc-chip-fill-color-default: rgba(mdc-theme-prop-value(on-surface), .12);
```

Comment:
These should all use the `mix()` function too, but that can go in another PR (to fix #2479)